src/omniperf

```diff
@@ -56,15 +59,24 @@ def verify_deps_version(localVer, desiredVer, operator):
 
 
 def verify_deps():
-    """Utility to read library dependencies from requirements.txt and endeavor to load them within current execution environment. 
-    Used in top-level omniperf to provide error messages if necessary dependencies are not available. """
+    """Utility to read library dependencies from requirements.txt and endeavor
+    to load them within current execution environment.
+    Used in top-level omniperf to provide error messages if necessary
+    dependencies are not available."""
+
+    # Check which version of python is being used
+    if sys.version_info[0] < 3 or sys.version_info[1] < 8:
```

Comment:
If there was ever a python 4, this could be problematic it's probably better to say:

```
if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] < 8):
```